mkt/extensions/migrations/0020_change_filename_scheme.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os.path
+
+from django.conf import settings
+from django.db import migrations
+
+from mkt.site.storage_utils import (move_stored_file, private_storage,
+                                    public_storage)
+
+
+def move_files_to_their_new_locations(apps, schema_editor):
```

Comment:
That's a fair question. I tested it locally manually, but it's super annoying to test migrations (they work on frozen models etc). If you can test it locally (with some existing extensionversions) that'd be cool